packages/service/src/ui-service/subscription/fake-subscription-service.ts

```diff
@@ -0,0 +1,30 @@
+import { Subscription } from "./subscription-models";
+import { SubscriptionService } from "./subscription-service";
+
+const subscriptions = [
+    { id: "/fake/sub1", displayName: "Subscription One" },
+    { id: "/fake/sub2", displayName: "Subscription Two" },
+    { id: "/fake/sub3", displayName: "Subscription Three" },
+    { id: "/fake/sub4", displayName: "Subscription Four" },
+    { id: "/fake/badsub", displayName: "Bad Subscription" },
+];
+
+export class FakeSubscriptionService implements SubscriptionService {
+    public list(): Promise<Subscription[]> {
+        return Promise.resolve(subscriptions as Subscription[]);
```

Comment:
Similar to the other comment, it doesn't really matter a whole lot but IMO `async` with a simple `return` statement is so nice syntactically.